src/frontend/src/heartbeat.rs

```diff
@@ -32,6 +33,22 @@ use crate::error::Result;
 
 pub mod handler;
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(default)]
+pub struct HeartBeatOptions {
```

Comment:
Also, I've noticed that you define 2 similar structs for `heartbeat`. I wonder if should we go with a single one?
nit: I would suggest the following renaming `HeartBeatOptions` -> `HeartbeatOptions`

Reply:
We can move `HeartBeatOptions` to `server` crate and keep two fields existing.

And let `frontend` and `datanode` use this option.

What do u think? @MichaelScofield